test/DynamoCoreWpfTests/app.config

```diff
@@ -1,11 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
-        <assemblyIdentity name="HelixToolkit.Wpf" publicKeyToken="52aa3500039caf0d" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-2015.1.595.0" newVersion="2015.1.595.0" />
+        <assemblyIdentity name="HelixToolkit.Wpf" publicKeyToken="52aa3500039caf0d" culture="neutral"/>
+        <bindingRedirect oldVersion="0.0.0.0-2015.1.595.0" newVersion="2015.1.595.0"/>
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
-</configuration>
\ No newline at end of file
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7"/></startup></configuration>
```

Comment:
Is this specific to `DynamoCoreWpfTests`?

Reply:
I think these get added to test assemblies because they are not referenced by any other assemblies, - it's kind of unnecessary as these are not app entry points AFAIK.